packages/firestore/src/local/indexeddb_index_manager.ts

```diff
@@ -131,7 +131,7 @@ export class IndexedDbIndexManager implements IndexManager {
     (l, r) => targetEquals(l, r)
   );
 
-  constructor(private user: User, private readonly databaseId: DatabaseId) {
```

Comment:
Can you share with me why `private` is removed?

Reply:
It's because the `User` argument is no longer stored in a property; instead, its "uid" is stored. Therefore, removing the `private` makes it no longer a property on the object but rather just an argument to the constructor.

Reply:
Thank you!